Modules/Messages/src/Messages.Domain/DomainEvents/Incoming/MessageOrphanedDomainEvent.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Backbone.BuildingBlocks.Domain.Events;
+
+namespace Backbone.Modules.Messages.Domain.DomainEvents.Incoming;
+
+public class MessageOrphanedDomainEvent : DomainEvent
+{
+    public required string IdentityAddress { get; set; }
```

Comment:
Here the property is called `IdentityAddress`, but in the outgoing event, it is called `CreatedBy`. So this will not work.

But this also tells me that neither you nor @MH321Productions have tested the code manually... 😠

Reply:
I swear to God that I have tested manually and it deleted the message, otherwise I wouldn't consider the PR ready for review.

Reply:
Maybe you renamed the property afterwards then? Because as it currently is, it should throw an exception when trying to deserialize the event on the receiving side (I haven't tested this, though, so I could be wrong)

Reply:
No property renaming was done, and no exception was thrown.

Reply:
Wow, interesting. I just did a small test, and it actually seems as if NewtonsoftJson doesn't throw if a `required` property does not exist. 😲 I really didn't expect that